### ISSUE TITLE
Defer entropy panel loading but don't load lazily

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/auspice#1899]
As part of performance improvements the entropy panel no longer loads when it is outside the viewport. It loads lazily, only when the panel comes into view.

To me it feels like a regression that now the entropy panel shows data loading for a split second when I scroll it into view.

Would it be possible to not load it lazily but instead paint it once the main work has been done?

Alternatively, the entropy panel could only be loaded lazily once the tree/view is above a certain size threshold so that the loading is only delayed when it really matters performance wise.

**Comment by jameshadfield** [comment]
> As part of performance improvements the entropy panel no longer loads when it is outside the viewport.

To be specific, the panel's still loaded but the data behind the bars isn't recomputed and the bars aren't repainted.

> it feels like a regression that now the entropy panel shows data loading for a split second when I scroll it into view

Yeah, this is a trade off (see #1879 for context). The delay you see here is time saved on essentially every single interaction in Auspice. (If you interact in a way which doesn't invalidate the entropy data then the entropy panel doesn't show "data loading".) I get that it's annoying to see "data loading" but I think it's better to have improved performance elsewhere.

> Would it be possible to not load it lazily but instead paint it once the main work has been done?

In short, yes. There's a few options to explore here (web workers, `requestIdleCallback`). Hopefully we can continue improving performance (there hasn't been much work here for a long time).
